io.github.alamahant.Asteria.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.alamahant.Asteria
+runtime: org.kde.Platform
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+command: Asteria
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+
+modules:
+  - name: python-flatlib
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps flatlib-0.2.3-py3-none-any.whl
```

Comment:
this would break if it is updated. unless you are planning to never update them? same for below.

Reply:
use https://github.com/flatpak/flatpak-builder-tools/tree/master/pip